recipes/aaplus/all/conanfile.py

```diff
@@ -56,44 +52,44 @@ def _compilers_minimum_version(self):
         }
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, "17")
+        if self.info.settings.compiler.cppstd:
```

Comment:
Why is get safe no longer needed?

Reply:
There is no get_safe with self.info.options

Reply:
see https://github.com/conan-io/conan-center-index/pull/12006#discussion_r939180538

Reply:
🤢 Thanks for the details

Reply:
Not yet: https://github.com/conan-io/conan/issues/11802